### DISCUSSION TITLE
科学上网 2.0 时代 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#26] [Announcements]
# 科学上网 2.0 时代 - forecho's Blog

引言
鉴于最近几年国内审查越来越严格，科学上网应该成为每个人的必备技能。本文将介绍我是如何科学上网的，以及我使用了哪些工具。
为什么要科学上网
科学上网的目的是为了访问被审查的网站，比如 Google、YouTube、Twitter、Telegram、Discord 等。这些网站在国内无法被访问，需要科学上网才能访问。
微信现在审查也越来越严格，群内聊天，你发敏感文字、图片只有你自己能看到，其他群友无法收到消息，真是特别恶心，所以我建议大家都使用 Telegram 吧。
YouTube 也是一个非常好的而且免费的资源平台，是除了 Google 之外我用的最多的网站了。

https://blog.forecho.com/science-internet-guide.html

**Comment by zhimiaoli** [comment]
1. clash for windows也有Mac版本，也叫这个名字，不过这个并非开源软件。
2. 还可以补充如何在路由器上配置，连上家里wifi就能自动科学。

**Reply by forecho** [reply]
1. 感谢更新，我已经更新
2. 这个我之前有写 https://blog.forecho.com/use-openclash.html

**Comment by tumutanzi** [comment]
点赞这类科普信息知识。
